src/components/TodoList/TodoList.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
```

Comment:
It's a good practice to use variables for colors, so they can be easily updated and reused. Consider adding a variable for the color used in `overflow: hidden;`.